cs/PerfTest/ParseArgs.cs

```diff
@@ -0,0 +1,225 @@
+                Console.WriteLine($"    {LogArg} <mode>: The disposition of the log after Upserts; default is {defaultTestResult.LogMode}");
+                Console.WriteLine($"        {LogMode.Flush.ToString()}: Copy entire log to disk, but retain tail of log in memory");
+                Console.WriteLine($"        {LogMode.FlushAndEvict.ToString()}: Move entire log to disk and eliminate data from memory as");
+                Console.WriteLine("             well. This will serve workload entirely from disk using read cache if enabled.");
+                Console.WriteLine("             This will *allow* future updates to the store.");
+                Console.WriteLine($"        {LogMode.DisposeFromMemory.ToString()}: move entire log to disk and eliminate data from memory as");
+                Console.WriteLine("             well. This will serve workload entirely from disk using read cache if enabled.");
+                Console.WriteLine("             This will *prevent* future updates to the store.");
+                Console.WriteLine($"    {ItersArg} <iters>: Number of iterations of the test; default = {defaultTestResult.IterationCount}");
+                Console.WriteLine();
+                Console.WriteLine($"  To compare result files (both options compare two JSON files containing {nameof(TestResults)}, where the difference is reported");
+                Console.WriteLine("             as 'second minus first', so if second does more operations per second, the difference is positive. If either of these");
+                Console.WriteLine("             is specified, no other non-Common parameters are allowed):");
+                Console.WriteLine($"    {CompareResultsExactArg} fnFirst fnSecond: Compare only {nameof(TestResults)} where the parameter specifications match exactly.");
+                Console.WriteLine("             Useful for comparing code change impact.");
+                Console.WriteLine($"    {CompareResultsSequenceArg} fnFirst fnSecond: Compare all {nameof(TestResults)} in sequence (to the length of the shorter sequence).");
+                Console.WriteLine("             Useful for comparing test-parameter change impact.");
+                Console.WriteLine();
+                Console.WriteLine("  Common parameters:");
+                Console.WriteLine($"    {ResultsFileArg} fname: The name of file to receive JSON text containing either:");
+                Console.WriteLine($"        {nameof(TestResultComparisons)}, if {CompareResultsExactArg} or {CompareResultsSequenceArg} was specified");
+                Console.WriteLine($"        {nameof(TestResults)} otherwise");
+                Console.WriteLine($"    {VerboseArg}: Verbose; print status messages; default = {verbose}");
+                Console.WriteLine($"    {PromptArg}: Print a message and wait before exiting; default = {prompt}");
+                Console.WriteLine();
+                return false;
```

Comment:
LogSettings has various settings that are configurable for the log, e.g., mutable fraction, total memory pages, segment size, etc. How do we control these in runs?